betterdiscordctl

```diff
@@ -238,103 +176,64 @@ while :; do
   shift
 done
 
-mkdir -p "$data"
-[[ -f $data/bd_map ]] || touch "$data/bd_map"
+# currently unused
+# mkdir -p "$data"
 
 # Commands
 
 bdc_status() {
+  asar_install=no
   index_mod=no
-  linked_dir=no
-  linked_repo=no
-  if [[ -d $core/injector ]]; then
-    if [[ -h $core/injector ]]; then
-      linked_dir=$(readlink "$core/injector")
-      if pushd "$core/injector" >/dev/null; then
-        linked_repo=$(git remote get-url origin 2>/dev/null || printf 'no\n')
-        popd >/dev/null
-      else
-        linked_dir="(broken link) $linked_dir"
-      fi
-    fi
-  fi
-  if [[ ! -f $core/index.js ]]; then
-    index_mod='(missing) no'
-  else
-    grep -q 'injector' "$core/index.js" && index_mod=yes
+  verbose 2 "VV: BetterDiscord asar: $bd_asar"
+  if [[ -h $bd_asar && ! -f $bd_asar ]]; then
+    asar_install='(broken link) no'
```

Comment:
I don't think there's a point in checking for links when we're not explicitly using them.